builtin/credential/approle/path_role.go

```diff
@@ -163,6 +167,11 @@ TTL will be set to the value of this parameter.`,
 					Type:        framework.TypeString,
 					Description: "Identifier of the role. Defaults to a UUID.",
 				},
+				"enable_local_secret_ids": &framework.FieldSchema{
```

Comment:
Could this just be `local_secret_ids` instead of with `enable` in front?

Reply:
It was `local_secret_ids` before. I changed it to better indicate that its a boolean and not expecting any secret IDs to be supplied. I don't have a strong preference to have `enable_` in the front. Let me know.

Reply:
Changed back to `local_secret_ids`.